### DISCUSSION TITLE
Failing to run tests  - Xcode Build failure

### DISCUSSION THREAD

**Discussion by brendonpomeroy** [leancodepl/patrol#1179] [Q&A]
Hi All,
I am trying to get started with patrol but my tests get stuck on ios-deploy and continuously print `Waiting for ios-deploy to launch the app is taking unusually long time (30.0s)`.

During the build step I get the following exception (but it doesn't stop the test):
```
2023-04-05 15:52:47.189 xcodebuild[80882:452273]  DVTAssertions: Warning in /System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot2/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/DVTiOSFrameworks/DVTiOSFrameworks-21506/DTDeviceKitBase/DTDKRemoteDeviceData.m:382
        Details:  (null) deviceType from 00008110-001919420A39401E was NULL when -platform called.
        Object:   <DTDKMobileDeviceToken: 0x12d62be60>
        Method:   -platform
        Thread:   <NSThread: 0x60000125b940>{number = 6, name = (null)}
        Please file a bug at https://feedbackassistant.apple.com with this warning message and any useful information you can provide.
```

I have followed the instructions in the native section of the docs (skipping the android section) to setup a test project (using only the auto generated flutter project) and this issue persists. When I cancel the cli command the device seems to launch the automation but does not do anything, it just sits on the home screen with the Automation running watermark.

Is anyone having a similar issue or know how I might rectify this issue?

**System Information**
 - Macbook Pro M1
 - MacOS Ventura 13
 - Xcode 14.2
 - Flutter 3.7.6
 - Android Studio 11.0.15

**Comment by bartekpacia** [comment]
Hi @brendonpomeroy, sorry for the problems - unfortunately it's [a known bug](#949). We currently don't have the resources to fix it now (we would gladly accept a PR though!), but it's on our radar and we _do_ want to fix it, soon.

**Comment by DimitarStoyanov91** [comment]
Hello,
Im am encountering the same problem.

**System Information**
Macbook Pro M1
MacOS Ventura 13
Xcode 14.2
Flutter 3.7.7
Android Studio 11.0.15
Patrol CLI 1.1.7
Patrol 1.0.9
